### PR TITLE
fix(grant): add 'isGrantInProgress' state for rendering after grant

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA

https://github.com/cloudforet-io/console/assets/83635051/70622d8e-97b3-40cd-a0e1-575100c40f47

### CODE REVIEW

**Review by yuda110 — APPROVED** [review]

File: apps/web/src/store/modules/user/actions.ts [hunk 1]
```diff
@@ -105,8 +105,7 @@ export const grantRole: Action<UserState, any> = async ({ commit, dispatch }, gr
 
             const roleInfo = await getGrantedRole(response.role_id, currentRoleType, response.role_type);
             commit('setCurrentRoleInfo', roleInfo);
-        }
-        commit('setCurrentRoleInfo', undefined);
+        } else commit('setCurrentRoleInfo', undefined);
```
Comment (yuda110) [inline apps/web/src/store/modules/user/actions.ts]: nitpick: please check comment

File: apps/web/src/store/modules/user/actions.ts [hunk 2]
```diff
@@ -87,23 +88,31 @@ const getRoleTypeFromToken = (token: string): RoleType => {
     const decodedToken = jwtDecode<JWTPayload>(token);
     return decodedToken.rol;
 };
-export const grantRole: Action<UserState, any> = async ({ commit }, grantRequest: Omit<TokenGrantParameters, 'grant_type'>) => {
+export const grantRole: Action<UserState, any> = async ({ commit, dispatch }, grantRequest: Omit<TokenGrantParameters, 'grant_type'>) => {
+    const fetcher = getCancellableFetcher(SpaceConnector.clientV2.identity.token.grant);
```
Comment (WANZARGEN) [inline apps/web/src/store/modules/user/actions.ts:92]:
```suggestion
    const fetcher = getCancellableFetcher<TokenGrantParameters, TokenGrantModel>(SpaceConnector.clientV2.identity.token.grant);
```

File: apps/web/src/store/modules/user/actions.ts [hunk 3]
```diff
@@ -87,23 +88,31 @@ const getRoleTypeFromToken = (token: string): RoleType => {
...
+
     try {
-        const response = await SpaceConnector.clientV2.identity.token.grant<TokenGrantParameters, TokenGrantModel>({
+        await dispatch('display/startGrantRole', undefined, { root: true });
+        const { status, response } = await fetcher<TokenGrantParameters, TokenGrantModel>({
```
Comment (WANZARGEN) [inline apps/web/src/store/modules/user/actions.ts:96]:
```suggestion
        const { status, response } = await fetcher({
```

**Review by WANZARGEN — APPROVED** [review]
LGTM~

**Pull request merged**